Snowflake.Data/Core/RestResponse.cs

```diff
@@ -167,10 +166,10 @@ internal class QueryExecResponseData
         internal Int64 returned { get; set; }
 
         [JsonProperty(PropertyName = "queryId", NullValueHandling = NullValueHandling.Ignore)]
-        internal string queryId { get; set; }
+        public string queryId { get; set; }
 
         [JsonProperty(PropertyName = "sqlState", NullValueHandling = NullValueHandling.Ignore)]
-        internal string sqlState { get; set; }
+        public string sqlState { get; set; }

```

Comment:
why to make it "public"?

Reply:
It's made public since it inherits from the IQueryExecResponseData interface now which has the queryId and sqlState members public